fenix/app/src/main/res/values/static_strings.xml

```diff
@@ -144,4 +144,10 @@
     <string name="translations_bottom_sheet_translating_in_progress">In progress</string>
     <string name="translations_bottom_sheet_translating_in_progress_content_description">Translating in Progress</string>
 
+    <string name="pref_never_translate_site_toolbar_title">Never translate these sites</string>
```

Comment:
Please also update this to `strings.xml` based on [this conversation](https://github.com/mozilla-mobile/firefox-android/pull/4226#discussion_r1406509320) on the other patch.

Reply:
Could you move the strings fully to `strings.xml`? Thanks for the naming update too!

Reply:
done